lib/components/atoms/FileInput/tests/FileInput.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { shallowWithTheme } from '../../../../../__mock__/themeMock';
+import { FileInput } from '../FileInput';
+
+describe('Input Component', () => {
+  let input = '';
+  const sampleProps = {
+    elementLocator: 'my-input-field',
+    name: 'my-input',
+    type: 'file',
+    onChange: () => {},
+  };
+
+  test('snapshot testing for input field', () => {
+    input = shallowWithTheme(<FileInput {...sampleProps} />);
+    expect(input).toMatchSnapshot();
+  });
+
+  test('should render file input component correctly', () => {
+    input = shallowWithTheme(<FileInput {...sampleProps} />);
+    expect(input.find('input').length).toBe(1);
+  });
+
+  test('should call function on click of the input field', () => {
+    function inputClicked(callback) {
+      callback();
+    }
+    input = shallowWithTheme(<FileInput {...sampleProps} />);
+    const changedEvt = jest.fn();
+    inputClicked(changedEvt);
+    expect(changedEvt).toHaveBeenCalledTimes(1);
+  });
+
+  test('should simulate click event of the input field', () => {
+    input = shallowWithTheme(<FileInput {...sampleProps} />);
+    const wrapper = input.simulate('click');
+    expect(wrapper.find('input').length).toBe(1);
+  });
+
+  test('should render with prop "multiple" to distinguish multiple or single file', () => {
```

Comment:
Move the tests related to type= file on the field itself

Reply:
This has been achieved.